ecs_fargate/README.md

```diff
@@ -180,6 +184,49 @@ Datadog's default CloudWatch crawler polls metrics once every 10 minutes. If you
 
 3. Finally, use a [Lambda function][22] to collect logs from CloudWatch and send them to Datadog.
 
+#### Fluent Bit and FireLens
+
+Configure the AWS FireLens integration built on Datadog's Fluent Bit output plugin to connect your FireLens monitored log data to Datadog Logs. 
+
+1. Enable the [Fluent Bit Datadog integration][27]. For more information, see the dedicated [Fluent Bit docs][27].
+2. Enable Fluent Bit in the FireLens log router container in your Fargate task. For more information about enabling FireLens, see the dedicated [AWS Firelens docs][28]. For more information about Fargate container definitons, see the [AWS docs on Container Definitions][26]. Here is an example snippet of a task definition where the Fluent Bit image is configured:
+    
+    ```
+    [ 
+      { 
+        "essential":true,
+        "image":"amazon/aws-for-fluent-bit:latest",
```

Comment:
let's mention that AWS recommends to use the regional docker image per https://github.com/aws/containers-roadmap/tree/master/preview-programs/firelens#using-the-aws-for-fluent-bit-image